app/jobs/regular/process_alert.rb

```diff
@@ -98,27 +98,16 @@ def create_new_topic(receiver, params, new_alerts)
           PluginStore.set(::DiscoursePrometheusAlertReceiver::PLUGIN_NAME, @token, receiver)
 
           assignee =
-            if params["commonAnnotations"]["topic_assignee"]
-              User.find_by(username: params["commonAnnotations"]["topic_assignee"])
-            elsif params["commonAnnotations"]["group_topic_assignee"]
-              random_group_member(params["commonAnnotations"]["group_topic_assignee"])
+            if username = params["commonAnnotations"]["topic_assignee"]
+              User.where("username_lower = ?", username.downcase).first
+            elsif group_name = params["commonAnnotations"]["topic_group_assignee"]
+              Group.where("LOWER(name) = ?", group_name.downcase).first
             end
 
           assign_alert(t, receiver, assignee: assignee)
         end
     end
 
-    def random_group_member(id_or_name)
```

Comment:
Dropping this as selecting a random group member is not a good thing to do ever.